soroban-sdk/src/env.rs

```diff
@@ -469,6 +469,79 @@ use std::{path::Path, rc::Rc};
 #[cfg(any(test, feature = "testutils"))]
 use xdr::{LedgerEntry, LedgerKey, LedgerKeyContractData, SorobanAuthorizationEntry};
 
+#[cfg(any(test, feature = "testutils"))]
+pub struct TestStellarAssetContractUtil {
+    pub address: Address,
+    env: Env,
+    issuer_id: xdr::AccountId,
```

Comment:
Could we add a function to expose the issuer address `address() -> Address` as well?

Reply:
Would this be helpful? `register_stellar_asset_contract_v2` updates the admin to the address passed in, so testing anything with the issuer account isn't really helpful.

Reply:
The utility isn't obvious, but given the StellarAssetIssuer type references an issue it feels appropriate to expose the address. The only use I can think of is someone wanting to access a StellarAssetIssuer in a snapshot they've loaded and so the symmetry makes sense.

Reply:
Done.